presto-hive/src/test/java/com/facebook/presto/hive/metastore/glue/TestHiveClientGlueMetastore.java

```diff
@@ -48,7 +48,7 @@ protected ExtendedHiveMetastore createMetastore(File tempDir)
         HdfsEnvironment hdfsEnvironment = new HdfsEnvironment(hdfsConfiguration, hiveClientConfig, new NoHdfsAuthentication());
         GlueHiveMetastoreConfig glueConfig = new GlueHiveMetastoreConfig();
         glueConfig.setDefaultWarehouseDir(tempDir.toURI().toString());
-
+        glueConfig.setMaxGlueConnections(10);
```

Comment:
What would happen if we continued to use default (5) here?

Reply:
Shouldn't matter. I can revert it if you would like